pom.xml

```diff
@@ -991,7 +991,7 @@ under the License.
 						<plugin>
 							<groupId>org.apache.maven.plugins</groupId>
 							<artifactId>maven-shade-plugin</artifactId>
-							<version>3.2.4</version>
+							<version>3.4.0</version>
```

Comment:
a) Why not 3.4.1?
b) Why make this specific to Java 11? We should try upgrading it in the default configuration and (I guess) get rid of the java11-specific configuration

FYI, 3.4.1 also fixes FLINK-20092 from some brief testing another person did.

Reply:
@rmetzger I'm the random person that @zentol mentioned!  

I was about to open a PR to bump this version to 3.4.1.

Reply:
a) no real reason. I thought it would be safer to update to the minimum required version.... but using the latest bugfix version should actually be even safer.
b) I didn't check the surroundings enough. Just looked for maven-shade-plugin and changed the version 🙈 

Great that this fixes more issues. Let's hope it works (my previous change was a no-op for our CI I guess ;) ) and it doesn't introduce other issues.